src/map/Map.js

```diff
@@ -611,6 +612,8 @@ L.Map = L.Evented.extend({
 		if (this.options.trackResize) {
 			L.DomEvent[onOff](window, 'resize', this._onResize, this);
 		}
+
+		this[onOff]('moveend', this._onMoveEnd);

```

Comment:
I don't know who would want to override this method — it's easier to just set false in options and then put your own listener for custom behavior. Not sure what you mean by `_leaflet_pos` changes.